docs/r3/example_design_for_atomic_swap.md

```diff
@@ -0,0 +1,212 @@
+# Design: Cross-Chain Atomic Swap between EVM and Corda Networks
+
+## Introduction
+This design document provides an outline of an atomic swap protocol for the exchange of assets between Ethereum Virtual Machine (EVM) based blockchains and Corda Networks, where the assets remain on their ‘home’ networks at all times. The protocol accommodates scenarios such as Delivery versus Payment (DvP), where the delivery on Corda can be completed (or reverted) if and only if the EVM payment is completed (or reverted), or Payment versus Payment (PvP) scenarios.
+
+It is intended as an example of a protocol that adopts the [architecture principles](architecture_principles.md) and [interoperability principles](../adhara/interop_principles.md), and respects the constraints, of regulated financial institutions, as elaborated in the documentation in the Harmonia repo.
+
+## Business Overview
+
+In what follows, we assume the model introduced in the [architecture documentation](beyondcandc.md). In particular, we assume the existence of two networks which, in this example, we will assume are a Digital Asset network on Corda and a digital cash network on Hyperledger Besu. Two participants, at some external venue, have agreed to a deal whereby one party will sell a digital asset to the other, in exchange for digital cash, and our job is to make this happen.
```

Comment:
The explanation makes sense, my issue would be that as described it's a "one-off" event between two parties who want to do the swap. In reality, we're talking about building solutions that encapsulate a problem requiring the swap as an ongoing operation

Reply:
Ah - ok.  I guess the point I was trying to make was that for any invocation of the atomic swap protocol we assume the parties have already agreed to the deal.  I take your point that an end-to-end solution may also include this negotiation but the key details of the protocol don't depend on it and so we can ignore it here.

Reply:
It might be enough to also add "We assume the creation of a business-level application that both exculpates the need for the exchange and initiates it on the appropriate chain"